templates/_macros/quote-wrapper.jinja

```diff
@@ -0,0 +1,138 @@
+{#
+  Params
+  - heading_text (string) (optional): The text to be displayed as the heading
+  - quote_heading_level (int) (optional): The heading level used for the quote text. Can be 2 or 4. Defaults to 2.
+  - quote_text (string) (required): The text of the quote. The macro will surround it with quotes, so there is no need to quote it yourself.
+  - citation_name (string) (optional): The name of the source being quoted
+  - citation_title (string) (optional): The title of the source being quoted
+  Slots
+  - signpost_image (optional): The signpost_image of the source being quoted
+  - heading_link (optional): Link to be displayed beside the heading text
+  - cta (optional): Contents of the call to action block to be displayed below the quote
+  - image (optional): An image to be displayed below the quote
+#}
+{%- macro quote_wrapper(
+  heading_text="",
+  quote_heading_level=2,
+  quote_text="",
+  citation_name="",
```

Comment:
I suppose it's possible that we could be quoting something said by multiple people or a larger organization, rather than a specific person. 

Maybe this could be `citation_source_name` for added clarity without discouraging people from citing people or larger organizations?

Reply:
Updated to `citation_source_name` and `citation_source_title`